### ISSUE TITLE
Better sync of u-turn customisations

### ISSUE THREAD

**Issue by aubergine10** [krzychu124/Cities-Skylines-Traffic-Manager-President-Edition#32]
There are three ways to control u-turns at junctions:

1. **Vehicles may do U-turns at junctions** mod setting (policies and restrictions tab)
2. **Junction restrictions**
3. **Lane connector**

There is some disconnect between 2 and 3.

* If I create a u-turn with **lane connector**, I can't remove it with **junction restrictions**
    * Junction restrictions detects that there is a u-turn but won't allow me to disable it via the icon
    * Desired behaviour: I should be able to disable a u-turn via junction restrictions (it would remove the relevant lane connections)
* If I create a u-turn with **junction restrictions**, there's no visualisation of that in the **lane connector**
    * Desired behaviour: The u-turn lane connection(s) should be visible in the lane connector

**Comment by krzychu124** [comment]
Yes, I've noticed that and I think its possible to do this but currently I have no idea how much work is needed to improve that.

**Comment by aubergine10** [comment]
As u-turn is essentially a lane arrow / connector thing, would it be worth merging in with those features? It would become part of #41 tool.
